src/chains/slide_gen_chain.py

```
@@ -0,0 +1,137 @@
+
+        for slide_param in slide_parameters:
+            function_name = slide_param.get("function_name")
+            parameters = slide_param.get("parameters", {})
+
+            func = self.slides_loader.get_function_by_name(template.id, function_name)
+
+            if func:
+                try:
+                    slide_content = func(**parameters)
+                    slides.append(slide_content)
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)